label_studio/core/label_config.py

```diff
@@ -261,7 +261,9 @@ def generate_sample_task_without_check(label_config, mode='upload', secure_mode=
 
         elif value == 'video' and p.tag == 'HyperText':
             task[value] = examples.get('$videoHack')
-
+        # List with a matching Ranker tag pair 
+        elif p.tag == 'List' and any([c.tag == 'Ranker' and c.get('toname') == p.get('name') or c.get('toName') == p.get('name') for c in xml.findall('.//Ranker')]):
```

Comment:
`List` tag has the same data regardless `Ranker` tag, the latest just allows it to be rearranged/groupped, so I think we can just check for `p.tag == 'List'` here

Reply:
Should be all updated now!